human-formatter/index.js

```diff
@@ -154,34 +156,44 @@ function prettyStackTrace (c, stack, basepath) {
       let func = match[1]
       let relative = match[2].slice(basepath.length)
       let converted = `${ PADDING }at ${ func } (./${ relative })`
+      // TODO: fix typo
       let isDependecy = match[2].includes('node_modules')
       return isDependecy ? c.gray(converted) : c.red(converted)
     }
   }).join(NEXT_LINE)
 }
 
-class HumanFormatter extends stream.Writable {
-  constructor (options) {
-    super()
-
-    if (typeof options.color === 'undefined') {
-      this.chalk = chalk
-    } else {
-      this.chalk = new chalk.Instance({ level: options.color ? 3 : 0 })
-    }
-
-    this.basepath = options.basepath || process.cwd()
-    this.out = options.out || process.stdout
-
-    if (this.basepath.slice(-1) !== path.sep) this.basepath += path.sep
+function humanFormatter (options) {
+  let c
+  if (options.color === undefined) {
+    c = chalk
+  } else {
+    c = new chalk.Instance({ level: options.color ? 3 : 0 })
   }
+  let basepath = options.basepath || process.cwd()
+  if (basepath.slice(-1) !== path.sep) basepath += path.sep
+  // Pino passes logger instance as this to prettifier constructor
+  // eslint-disable-next-line no-invalid-this
```

Comment:
Add override to `package.json`→`eslintConfig` to disable `no-invalid-this` for this file